types/comma-number/comma-number-tests.ts

```diff
@@ -0,0 +1,11 @@
+import commaNumber from 'comma-number';
```

Comment:
```suggestion
import commaNumber = require('comma-number');
```

Reply:
you have to change this import to use require (or use old syntax 'import * as commaNumber from ....'